src/exit.rs

```diff
@@ -45,8 +46,19 @@ impl From<AccessWidth> for usize {
 type Port = u16;
 
 /// The result of [`AxArchVCpu::run`].
+/// Can we reference or directly reuse content from [kvm-ioctls](https://github.com/rust-vmm/kvm-ioctls/blob/main/src/ioctls/vcpu.rs) ?
 #[non_exhaustive]
-pub enum AxArchVCpuExitReason {
+#[derive(Debug)]
+pub enum AxVCpuExitReason {
+    /// The instruction executed by the vcpu performs a hypercall.
+    Hypercall {
+        /// The hypercall number.
+        nr: u64,
+        /// The arguments for the hypercall.
+        args: [u64; 6],
+        /// Whether the hypercall was executed in long mode.
+        longmode: u32,
```

Comment:
Is this field necessary? If so, can we make it a `bool`?

Reply:
Actually, both `HyperCall` and `FailEntry` was copied from [kvm-ioctls](https://github.com/rust-vmm/kvm-ioctls/blob/main/src/ioctls/vcpu.rs)

Reply:
Copying without rustifying seems less convincing🙂